src/button/button-group.scss

```diff
@@ -6,45 +6,53 @@
   display: inline-flex;
   align-items: center;
 
-  > .iui-button {
+  $childSelector: '&, input, button'; // target nested inputs and buttons
```

Comment:
nested inputs need to be targeted because inputs can be inside `iui-input-container`, and disabled buttons can be inside a wrapper.

Reply:
we want to target native html elements or only itwinui ones?

Reply:
> we want to target native html elements or only itwinui ones?

Not sure what you mean. itwinui components use the native elements. I am targeting `input`/`button` instead of classes to keep the specificity low